microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-105.md

```diff
@@ -0,0 +1,148 @@
+---
+title: What's new in DevTools (Microsoft Edge 105)
+description: "Focus Mode: Improved location controls for DevTools, Activity Bar, Quick View, and Improved reliability in Issues tool. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 09/01/2022
+---
+# What's New in DevTools (Microsoft Edge 105)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+<!-- ====================================================================== -->
+## Test your PWA's protocol handlers from the Application tool  
+
+<!-- Rank: 1 -->
+<!-- Title: Test your PWA's protocol handlers from the Application tool  -->
+<!-- Subtitle: From the Manifest section of the Application tool, you can now provide custom protocols to launch your PWA. -->
+
+In Microsoft Edge 105, the **Application** tool now supports testing protocol handlers. Since Microsoft Edge 96, you've been able to define protocol handlers in the application manifest of your Progressive Web App (PWA). Now you can  test those protocols from the **Application** tool itself in DevTools if you have your PWA installed. 
+
+With DevTools open against your PWA:
+1. Open the **Application** tool.
+2. Expand the Manifest dropdown.
+3. Select the 'Protocol Handlers' entry in the dropdown.
+4. Select the protocol you want to test from the dropdown.
+5. Enter in the URL or endpoint that you would like to test.
+6. Select **Test Protocol**.
+
+The **Application** tool will attempt to launch your PWA with the protocol and URL you specified. The browser asks for permission to open the application, and then prompts you to confirm that you want to handle the protocol and the app you'd like to launch. If you give permission, your app will open with the specified content.
+
+![Protocol Handler](./devtools-105-images/protocol-handler.png)
```

Comment:
For this image, can you type in something to the textbox, like "americano" so it's clear that you're testing a protocol? In the current screenshot, the textbox is blank